### PR TITLE
Add android_views integration tests to ci.yaml

### PR DESCRIPTION
**Pull request by GaryQian**
Adds a builder to run the android recipe introduced in https://flutter-review.googlesource.com/c/recipes/+/16580

### CODE REVIEW

File: .ci.yaml [hunk 1]
```diff
@@ -1676,6 +1676,18 @@ targets:
...
+  - name: Linux android views
+    recipe: flutter/android_views
+    bringup: true
+    properties:
+      dependencies: >-
+        [
+          {"dependency": "android_sdk"},
+          {"dependency": "android_avd", "version": "31"}
+        ]
+    timeout: 60
```
Comment (CaseyHillers) [inline .ci.yaml:1688]: Does this need 60 minutes?

**Review by CaseyHillers — APPROVED** [review]

File: .ci.yaml (hunk 1 above)
Comment (GaryQian) [reply .ci.yaml:1688]: Probably not, but it also seems that every other test has the same 60min timeout.

**Review by blasten — APPROVED** [review]
LGTM!

**Comment by keyonghan** [comment]
Can we add testownership in [`TESTOWNERS`](https://github.com/flutter/flutter/blob/master/TESTOWNERS)? This is causing post-submit test `Linux test_ownership` failure:
Screen Shot 2021-08-23 at 6 10 21 PM

@CaseyHillers seems we need a way to enable pre-submit test when it is marked as flaky in post-submit.

**Comment by CaseyHillers** [comment]
@keyonghan why is TESTOWNERS marked `bringup: true`?

**Comment by keyonghan** [comment]
> @keyonghan why is TESTOWNERS marked `bringup: true`?

The is a newly added test, and is waiting for the auto-process to pick it up.
@chunhtai Did a quick check, seems `Linux test_ownership` has passed for recent 50 runs on Monday (failure started yesterday). Could you help check why it was not auto marked as non-flaky?

**Comment by chunhtai** [comment]
@keyonghan the deflake handler is out of commission until this pr is merged https://github.com/flutter/cocoon/pull/1335. I have not looked into why the waiting for tree label does not work

**Comment by GaryQian** [comment]
https://github.com/flutter/flutter/pull/88797 adds a TESTOWNERS entry

**Pull request merged**